Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -227,19 +235,21 @@ - (void)startRequest:(BTPaymentFlowRequest *)request configuration:(BTConfigurat
 }
 
 - (void)processLookupResult:(BTThreeDSecureResult *)lookupResult configuration:(BTConfiguration *)configuration {
-    if (!lookupResult.lookup.requiresUserAuthentication) {
-        [self.paymentFlowDriverDelegate onPaymentComplete:lookupResult error:nil];
+    if (!lookupResult.lookup.isThreeDSecureVersion2) {
+        NSError *error = [NSError errorWithDomain:BTThreeDSecureFlowErrorDomain
+                                               code:BTThreeDSecureFlowErrorTypeConfiguration
+                                           userInfo:@{NSLocalizedDescriptionKey: @"3D Secure v1 is deprecated and no longer supported. See https://developer.paypal.com/braintree/docs/guides/3d-secure/client-side for more information."}];
+        
+        [self.paymentFlowDriverDelegate onPaymentComplete:nil error:error];
         return;
     }
     
-    if (lookupResult.lookup.isThreeDSecureVersion2) {
```

Comment:
q: Do any tests need to be changed for this? I noticed we're using `requiresUserAuthentication` now.

Reply:
We do both. Just above this we do `if (!lookupResult.lookup.isThreeDSecureVersion2)`